static/app/views/settings/components/teamSelect.tsx

```diff
@@ -92,8 +92,13 @@ function TeamSelect({
   };
 
   // Only show options that aren't selected in the dropdown
+  // and that aren't idp-provisioned
   const options = teams
-    .filter(team => !selectedTeams.some(selectedTeam => selectedTeam.slug === team.slug))
+    .filter(
+      team =>
+        !selectedTeams.some(selectedTeam => selectedTeam.slug === team.slug) &&
+        !team.idpProvisioned
```

Comment:
I don't think we should filter out IdP-provisioned in this dropdown. 

As a user, it would be frustrating that I cannot find a team in this dropdown and I wouldn't know that it was not included because it was IdP-provisioned. IMO they should be included, but disabled and labeled differently.

Reply:
i made them grayscale and with a tooltip with the template text if they're idpProvisioned